lambda/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "lambda"
```

Comment:
can we have more specific name, related to aws?

Reply:
renamed to `aws-lambda`